modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -59,7 +60,8 @@ def validate_veteran_identifiers(require_birls: false) # rubocop:disable Metrics
         ClaimsApi::Logger.log('validate_identifiers',
                               rid: request.request_id, mpi_res_ok: mpi_add_response.ok?,
                               ptcpnt_id: target_veteran.participant_id.present?,
-                              birls_id: target_veteran.birls_id.present?)
+                              birls_id: target_veteran.birls_id.present?,
+                              icn: target_veteran.mpi[:user_identity][:icn])
```

Comment:
are we always going to have this value? we'll get a no method [] error if not. If it's a veteran request we have access as `target_veteran.icn` but I'm not sure about VSO / representative. I'll have to look at this more tomorrow to see.